libqcow.yaml

```diff
@@ -0,0 +1,62 @@
+package:
+  name: libqcow
+  version: "20240308"
+  epoch: 1
```

Comment:
epoch should be 0 for new packages but given it's in now, we can keep it as it is.